vendor/k8s.io/kubernetes/pkg/kubectl/cmd/util/helpers.go

```diff
@@ -154,7 +155,13 @@ func checkErr(err error, handleErr func(string)) {
 
 func statusCausesToAggrError(scs []unversioned.StatusCause) utilerrors.Aggregate {
 	errs := make([]error, len(scs))
+	errorMsgs := sets.NewString()
 	for i, sc := range scs {
+		// check for duplicate error messages and skip them
+		if errorMsgs.Has(sc.Message) {
+			continue
```

Comment:
what you want to do is allocate errs with capacity of len(scs), but length 0, then use `errs = append(errs, ...)` instead of assigning directly to indexes